### PR TITLE
Get rid of StixObjectIdEncoder

### PR DESCRIPTION
**Pull request by baulus**

### CODE REVIEW

File: stix_shifter_utils/utils/helpers.py [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
-            return obj.object_id
-        # Let the base class default method raise the TypeError
-        return json.JSONEncoder.default(self, obj)
+import json_fix
 
 
 class StixObjectId(object):
     object_id:str = None
     def __init__(self, object_id):
+        json_fix
```
Comment (mdazam1942) [inline stix_shifter_utils/utils/helpers.py:8]: any initialization value `json_fix` == ?

Comment (baulus) [reply stix_shifter_utils/utils/helpers.py:8]: yes, it is a hack to make EDI and other developer (not to delete it when EDI suggests it is not used) to think it is used somehow, not just `import json_fix`
in reality just import json_fix is enough

Comment (baulus) [reply stix_shifter_utils/utils/helpers.py]: having a second thought on this, I think you are right, any developer that sees this hack will think it is not initialized variable
removed, added comment next to import

**Review by delliott90 — APPROVED** [review]

**Comment by baulus** [comment]
Fix link: https://stackoverflow.com/questions/3768895/how-to-make-a-class-json-serializable#68926979
Lib: https://pypi.org/project/json-fix/#description

**Review by mdazam1942 — APPROVED** [review]

**Pull request merged**